packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/test.sh

```diff
@@ -16,6 +16,47 @@ test_val "$(compare_swing_store_export_data $EXPORT_DIR)" "match" "swing-store c
 rm -rf $EXPORT_DIR
 startAgd
 
+testHasChildren() {
+    path=$1
+    min=$2
+    line="$(agd query vstorage children $path -o jsonlines)"
+    ok=$(echo $line | jq ".children | length | . > $min")
+    test_val "$ok" "true" "$path: more than $min children"
+}
+
+# Check brand aux data for more than just vbank assets
+testHasChildren published.boardAux 3
+
+testDisplayInfo() {
+    name=$1
+    expected=$2
+
+    line="$(agoric follow -lF :published.agoricNames.brand -o text)"
+    # find brand by name, then corresponding slot
+    id=$(echo $line | jq -r '.slots as $slots | .body | gsub("^#";"") | fromjson | map(.[0]) | index("'$name'") | $slots[.]')
```

Comment:
It's probably better to use actual slot indexes than assuming alignment of array positions.
```suggestion
    id=$(echo $line | jq --arg name "$name" -r '.slots as $slots | .body | gsub("^#";"") | fromjson | .[] | select(.[0] == $name) | .[1] | capture("^[$](?<slot>0|[1-9][0-9]*)") | .slot | $slots[. | tonumber]')
```

Reply:
We're migrating from bash to js, so I didn't think it was worth figuring out that level of complexity in the jq language. But now that you figured it out, yes, it is better. So thanks.

Reply:
Agree on both counts